src/components/list/list-item.types.ts

```diff
@@ -45,6 +45,11 @@ export interface ListItem<T = any> {
      * Component used to render in the list
      */
     primaryComponent?: ListComponent;
+
+    /**
+     * True if the list item should be removable (only works for limel-chips)
+     */
+    removeable?: boolean;
```

Comment:
I'm not sure this makes much sense at the moment from the lists point of view, so I don't think we should add this here. Also, I'm not sure about the description, how is this connected to the chips? It seems to be used by the picker and the `Chip` interface :thinking:

Reply:
Ah good point! I double checked, it seems we could instead pass in the `removable` (I misspelled it in the commit 💀 ) as a prop into the `limel-picker` in `relation-picker.tsx`. The only reason we need it is for `createChip()` in `picker.tsx`, and I just blindly added it to the interface

Reply:
Looked into it more

We can close this PR, all the changes can be handled by just passing in`readonly=true`